src/lib/scanner.ts

```diff
@@ -98,20 +102,11 @@ export class ScanRunner {
      * Creates the arguments for an execution of {@code sf scanner run}.
      * @param targets The files to be scanned.
      */
-    private async createPathlessArgArray(targets: string[]): Promise<string[]> {
-        const args: string[] = [
-            'scanner', 'run',
-            '--target', `${targets.join(',')}`,
-            '--engine', 'pmd,retire-js',
-            '--json'
-        ];
-        const customPmdConfig: string = SettingsManager.getPmdCustomConfigFile();
-        // If there's a non-null, non-empty PMD config file specified, use it.
-        if (customPmdConfig && customPmdConfig.length > 0) {
-            if (!(await exists(customPmdConfig))) {
-                throw new Error(messages.error.pmdConfigNotFoundGenerator(customPmdConfig));
-            }
-            args.push('--pmdconfig', customPmdConfig);
```

Comment:
What happened to this customPmdConfig? Why did it get dropped?

Reply:
This was deleted by mistake. I added this back in.